docs/guides/http-filters.md

```diff
@@ -102,7 +102,7 @@ The request object has the following shape:
 ```typescript
 type Request = {
   method: string
-  url: string
+  uri: string
```

Comment:
uri is not a string any more. Please check again.

Reply:
done